CODEOWNERS

```diff
@@ -156,7 +156,7 @@
 /specification/monitor/ @gucalder
 
 # PRLabel: %Network
-/specification/network/ @MikhailTryakhov
+/specification/network/ @Joanna-Yang-Art
```

Comment:
This user doesn't have write permissions to the repo so github will not add them as reviewers. They will need to get the correct permissions before this will be helpful.

Reply:
@weshaggard how to get these permissions for @Joanna-Yang-Art? The main idea from me - I definitely don't need to be an owner anymore...

Reply:
OK @MikhailTryakhov I'll go ahead and merge this to get you off the reviewer set.

@Joanna-Yang-Art please see https://aka.ms/azsdk/permission for the necessary permissions.